AdyenComponents/BasicPersonalInfoFormComponent/BasicPersonalInfoFormComponent.swift

```diff
@@ -9,6 +9,8 @@ import Foundation
 import UIKit
 
 /// A component that provides a form consisting of first name, last name, phone, and email.
+/// Used for E-context (ATM, Store, Online) or 7eleven payment methods.
+/// :nodoc:
```

Comment:
The first line is enough, `BasicPersonalInfoFormComponent` is not necessarily for `E-context (ATM, Store, Online) or 7eleven payment methods`

Reply:
👍